frontend/src/views/camp/Material.vue

```diff
@@ -1,145 +1,41 @@
 <!--
-Admin screen of a camp: Displays MaterialLists and MaterialItems
+Show all material lists for a camp on mobile
 -->
 
 <template>
   <content-card :title="$tc('views.camp.material.title')" toolbar>
     <template #title-actions>
-      <v-btn small color="primary" class="ml-5" @click="downloadXlsx">
-        <v-icon left>mdi-microsoft-excel</v-icon>
-        {{ $tc('views.camp.material.downloadXlsx') }}
-      </v-btn>
+      <DialogMaterialListCreate :camp="camp()">
```

Comment:
Here you have to check if the user is guest

Reply:
[bf9e996](https://github.com/ecamp/ecamp3/pull/3797/commits/bf9e996b56ef5574d91aae50cdb9388a31b266bf)